src/qt/createwalletdialog.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_QT_CREATEWALLETDIALOGG_H
+#define BITCOIN_QT_CREATEWALLETDIALOGG_H
+
+#include <qt/walletcontroller.h>
+
+#include <QDialog>
+
+class BitcoinGUI;
+class WalletModel;
+
+namespace Ui {
+    class CreateWalletDialog;
+}
+
+/** Dialog for creating wallets
+ */
+class CreateWalletDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit CreateWalletDialog(QWidget *parent);
+    ~CreateWalletDialog();
+
+    void accept();
+
+    void SetWalletController(WalletController *wallet_controller);
```

Comment:
Why is this needed as a separate function? Can the `WalletController` just be passed as an argument to the constructor?

Reply:
Done